tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml

```diff
@@ -14,10 +14,10 @@
 ---
 
 test_name: hpc-slurm-chromedesktop
-deployment_name: "enter-{{ build }}"
+deployment_name: "slurm-crd-{{ build }}"
 # Manually adding the slurm_cluster_name for use in node names, which filters
 # non-alphanumeric chars and is capped at 10 chars.
-slurm_cluster_name: "enter{{ build[0:5] }}"
+slurm_cluster_name: "crd{{ build[0:5] }}"
```

Comment:
nit: would be good to change this to `slurm-crd{{ build[0:5] }}` for consistency.

Reply:
This was intentional, slurm will only use a few characters from the name (like 6 or maybe 8) when naming resources so if we use `slurm-crd-{build}` the build id will not make it into instance names. It will be clear that the instances are for slurm because the will append other identifiers like "controller".

Reply:
Infact `slurm_cluster_name` must be a subset of the deployment name. Test caught it. I am rerunning with an update.